client/db_test.go

```diff
@@ -266,7 +266,7 @@ func ExampleTx_Commit() {
 	// 1/0: ab=2
 }
 
-func ExampleDB_Insecure() {
+func ExampleDB_second() {
```

Comment:
Looks like the schema is [`Object`](https://godoc.org/go/types#Object)_`Object`(returned from [`LookupFieldOrMethod`](`https://godoc.org/go/types#LookupFieldOrMethod`))_startsWithLowercaseSuffix